bin/load-mocks

```diff
@@ -662,7 +689,7 @@ def main(num_events=1, extra_events=False):
 
             if hasattr(buffer, "process_pending"):
                 print ("    > Processing pending buffers")  # NOQA
-                buffer.process_pending()
+                print ("    > Processing pending buffers")  # NOQA
```

Comment:
What's this change?

Reply:
hmmr term may have been bugging out, will fix